### PR TITLE
refs[TASK][31136]: UI: Matches Screen

### PR DESCRIPTION
**Pull request by Domhoan**
## Related Tickets
- [#Task 31136: UI: Matches Screen](https://edu-redmine.sun-asterisk.vn/issues/31136)
## WHAT
- UI: Matches Screen
## HOW
- App theme & bottom navigation
- ActionBar: title , icon search_match
- Layout fragment match: recyclerView for match item
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/fragment_match.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerViewMatch"
+        android:layout_width="match_parent"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_match.xml]: dùng 0dp hoặc match constraint

File: app/src/main/res/layout/item_recyclerview_match.xml [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/textDate"
+        android:layout_width="match_parent"
```
Comment (thonc-2135) [inline app/src/main/res/layout/item_recyclerview_match.xml]: tương tự như trên

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <FrameLayout
+        android:id="@+id/fragmentContainer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```
Comment (huytq-2422) [inline app/src/main/res/layout/activity_main.xml]: ConstraintLayout thì dùng match_contraint thay vì match_parent
```suggestion
        android:id="@+id/fragmentContainer"
        android:layout_width="0dp"
        android:layout_height="0dp"
```

File: app/src/main/res/layout/fragment_match.xml (hunk 1 above)
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_match.xml]:
```suggestion
    <androidx.recyclerview.widget.RecyclerView
        android:id="@+id/recyclerMatches"
        android:layout_width="0dp"
```

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <dimen name="dp_80">80dp</dimen>
+    <dimen name="dp_24">24dp</dimen>
+    <dimen name="dp_32">32dp</dimen>
+    <dimen name="dp_48">48dp</dimen>
+    <dimen name="sp_24">24sp</dimen>
+    <dimen name="sp_20">20sp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="sp_16">16sp</dimen>
```
Comment (thonc-2135) [inline app/src/main/res/values/dimens.xml]: sắp xếp theo thứ tự , tách sp vs dp ra

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="app_name">Premier League</string>
     <string name="err_fist_param_not_be_null">First param should not be null, or you forgot passing one</string>
+    <string name="title_tran_dau">TRẬN ĐẤU</string>
```
Comment (thonc-2135) [inline app/src/main/res/values/strings.xml]: tiếng anh tiếng việt rõ ràng

File: app/src/main/res/layout/item_recyclerview_match.xml [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+        android:src="@drawable/ic_bell"
+        app:layout_constraintEnd_toEndOf="@+id/textTime"
+        app:layout_constraintStart_toStartOf="@+id/textTime"
+        app:layout_constraintTop_toBottomOf="@+id/textTime" />
+
+    <ImageView
+        android:id="@+id/imageAwayTeam"
+        android:layout_width="@dimen/dp_48"
+        android:layout_height="@dimen/dp_48"
+        android:layout_marginStart="16dp"
```
Comment (huytq-2422) [inline app/src/main/res/layout/item_recyclerview_match.xml]: hardcode

File: app/src/main/res/menu/menu_bottom_navigation.xml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+    <item
+        android:id="@+id/menuStat"
+        android:icon="@drawable/ic_stat"
+        android:title="Stat" />
+
+    <item
+        android:id="@+id/menuSetting"
+        android:icon="@drawable/ic_settings"
+        android:title="Setting" />
```
Comment (thonc-2135) [inline app/src/main/res/menu/menu_bottom_navigation.xml]: hard code

File: app/src/main/res/layout/item_recyclerview_match.xml [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+        app:layout_constraintVertical_bias="0.0" />
+
+    <TextView
+        android:id="@+id/textHomeTeam"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dp_16"
+        android:layout_marginEnd="@dimen/dp_8"
+        android:ellipsize="end"
+        android:gravity="left"
```
Comment (huytq-2422) [inline app/src/main/res/layout/item_recyclerview_match.xml]:
```suggestion
        android:gravity="start"
```

File: app/src/main/res/values/dimens.xml [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+</resources>
```
Comment (huytq-2422) [inline app/src/main/res/values/dimens.xml]: Sắp xếp, phân loại nhé

File: app/src/main/res/layout/item_recyclerview_match.xml [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+        app:layout_constraintBottom_toBottomOf="@+id/imageAwayTeam"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/imageAwayTeam"
+        app:layout_constraintTop_toTopOf="@+id/imageAwayTeam" />
+
+    <ImageView
+        android:id="@+id/imageHomeTeam"
+        android:layout_width="@dimen/dp_48"
+        android:layout_height="@dimen/dp_48"
+        android:layout_marginEnd="16dp"
```
Comment (thonc-2135) [inline app/src/main/res/layout/item_recyclerview_match.xml]: hard code

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (huytq-2422) [inline app/src/main/res/values/strings.xml]: Sao lại strings.xml lại có cả tiếng anh và tiếng việt thế
```suggestion
    <string name="title_match">TRẬN ĐẤU</string>
```

File: app/src/main/AndroidManifest.xml [hunk 11]
```diff
@@ -15,14 +15,15 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".ui.splash.SplashActivity"
+            android:label="@string/title_tran_dau"
```
Comment (tuannm-1276) [inline app/src/main/AndroidManifest.xml]: co gi do sai sai nhi

File: app/src/main/res/layout/activity_main.xml [hunk 12]
```diff
@@ -1,18 +1,26 @@
...
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toTopOf="@+id/bottomNavigationView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <com.google.android.material.bottomnavigation.BottomNavigationView
+        android:id="@+id/bottomNavigationView"
+        android:layout_width="match_parent"
```
Comment (tuannm-1276) [inline app/src/main/res/layout/activity_main.xml]: lại sai rồi

File: app/src/main/res/values/strings.xml [hunk 13]
```diff
@@ -1,4 +1,8 @@
 <resources>
     <string name="app_name">Premier League</string>
     <string name="err_fist_param_not_be_null">First param should not be null, or you forgot passing one</string>
+    <string name="title_tran_dau">MATCHES</string>
```
Comment (thonc-2135) [inline app/src/main/res/values/strings.xml]: lại tiếng việt r

**Pull request merged**